README.md

```diff
@@ -86,6 +87,14 @@ Links:
 Procedure:
 *TBA*
 
+Current Release Cycle:
+
+[Dates and further information for the 1.22 Release](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.22)
+
+Exceptions to Enhancement Milestone Dates:
+
+The Exceptions Process is handled by the Release Team, please see their [documentation](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md) for further details.
+
```

Comment:
small nit: can we add the same sections to the Table of Contents at the top of the doc?

Reply:
just to clarify you want these sections to become large section headers in the body of the doc (basically level with Enhancements Tracking Spreadsheet)?

Reply:
ping @palnabarun

Reply:
@kikisdeliveryservice -- we can use L3 headers ### for the "Current release cycle" and L2 ## for "Exceptions to Enhancement Milestone Dates".

Thanks for the ping @annajung :)

Reply:
done